ajax/libs/jquery-easy-ticker/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery-easy-ticker",
+  "filename": "jquery.easy-ticker.min.js",
+  "version": "2.0.0",
+  "description": "jQuery Easy ticker plugin",
+  "homepage": "http://www.aakashweb.com/jquery-plugins/easy-ticker/",
+  "keywords": [
+    "jquery",
+    "ticker",
+    "easy",
+    "animation",
+    "slider",
+    "effect",
+    "ticker",
+    "news",
+    "rss",
+    "feed",
+    "vertical",
+    "easing"
+  ],
+  "repository": {
+    "type": "plain file",
+    "url": "https://github.com/vaakash/jquery-easy-ticker/blob/master/jquery.easy-ticker.min.js"
+  },
```

Comment:
Repository is wrong, please take a look at other libraries like `jqurey`, thanks.